src/components/RubricsIndex.js

```diff
@@ -84,4 +79,4 @@ const mapDispatchToProps = (dispatch) => {
   return bindActionCreators(Actions, dispatch);
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(RubricsIndex);
+export default withRouter(connect(mapStateToProps, mapDispatchToProps)(RubricsIndex));
```

Comment:
I see. The basic idea is that we now need to find a way to inject the routing props to the child component.  I see that we are wrapping the child in a HOC withRouter.